src/main/kotlin/gomushin/backend/alarm/service/QuestionAlarmService.kt

```
@@ -0,0 +1,47 @@
+    @Scheduled(cron = "0 0 19 * * *", zone = "Asia/Seoul")
+    fun sendQuestionAlarms() {
+        val coupleMembers = memberService.getAllCoupledMember()
+        runBlocking {
+            coupleMembers.map { member ->
+                async(Dispatchers.IO) {
+                    try {
+                        val notificationContent = questionMessages.random()
+                        val (title, sendContent) = MessageParsingUtil.parse(notificationContent)
+                        log.info("질문형 메시지 전송 : 수신자 {${member.name}}, 제목 {${title}}, 내용{${sendContent}}, 전송시각{${LocalDateTime.now()}}\n")
+                        fcmService.sendMessageTo(member.fcmToken, title, sendContent)
+                    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)